core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -40,6 +48,22 @@ function toTitleCase(str) {
     });
 }
 
+type Actions = {
+    command: string;
+    menuLabel: string;
+    icon: string;
+    name: string;
+};
+
+type Block = {
+    url: string;
+    actions: Array<Actions>;
+};
+
+type CustomBlock = {
+    extensions: Block[];
+};
+
```

Comment:
We can move these types to this [file](https://github.com/dotCMS/core/blob/0dea6164a7ee2ab47a5c3bcce49a76aed8776812/core-web/libs/dotcms-models/src/lib/dot-block-editor.model.ts)